assets/terraform-modules/packet/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -131,6 +131,9 @@ systemd:
           --client-ca-file=/etc/kubernetes/ca.crt \
           --cluster_dns=${k8s_dns_service_ip} \
           --cluster_domain=${cluster_domain_suffix} \
+          %{~ if cloud_provider == "external" ~}
```

Comment:
Actually, we don't really need a variable for it. I think we can just hardcode external provider here.